DDUFSL/XmlManager/XmlManager.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection.Metadata;
+using System.Text;
+using System.Threading.Tasks;
+using System.Xml;
+using System.Xml.Linq;
+
+namespace DDUFSL.XmlManager
+{
+    /// <summary>
+    /// A 'config.xml' file-t az executable mellé kell rakni
+    /// 
+    /// Egy példa struktúra:
+    /// 
+    /// <FilterConfig>
+    ///     <Folder>C:\Users\GreG\Downloads\Qpak-server\mods</Folder>   //A mappa útvonala amit rendezni szeretnél
+    ///     <FileTypes> //FileTípusok
```

Comment:
Kód helett pl ezt kiírhatta volna egy config.xml.sample fájlba is, hogy a user tudjon magának csinálni a kód megnézése nélkül.